### PR TITLE
Create cycan project

### PR DESCRIPTION
**Pull request by derrickf87**
Add standard info

### CODE REVIEW

File: projects/cycan/tags [hunk 1]
```diff
@@ -0,0 +1 @@
+NFT,DeFi,SmartContracts,Exchange,Dex
```
Comment (santryhuang) [inline projects/cycan/tags:1]: please select the one（If you choose too much, the interface will be messy）

File: projects/cycan/README_zh.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+## 项目介绍
+
+Cycan，意为Cyber Canaan，即加密数字世界的迦南之地。
+Cycan network（CYN）是跨链去中心化资管平台，是基于Polkadot网络的平行链，其先行链 Everlasting parachain（ELP）是基于Kusama的平行链，目标是构建基于WASM技术的去中心化资产管理协议，让每个人都可以自由、安全、高效地在链上发起、管理和投资去中心化基金。
+Cycan还开发了一种采用“储备+算法”双型且具备抗通胀特性的Stablecoin，即Everlasting Cash（ELC）。未来，Cycan将与其他平行链合作通过XCMP协议迅速实现DeFi乐高协作，也将通过异构跨链桥，与ETH、BSC、SOL等区块链网络间实现去中心化加密资管业务的跨链互通。
+
+## Wasm合约进展
+
+Cycan 已将ELC(Everlasting Cash)项目提交 Web3 基金会申请Open Grant，并成为第九批Grant项目之一，目前已完成里程碑1开发成果并提交审核中，同时正在全力开发里程碑2内容。[查看ELC项目代码](https://github.com/CycanTech/ELC)
+
```
Comment (santryhuang) [inline projects/cycan/README_zh.md:12]: Could you describe Wasm technology as much as possible? or Wasm roadmap etc.

File: projects/cycan/tags (hunk 1 above)
Comment (derrickf87) [reply projects/cycan/tags]: Solved

File: projects/cycan/README_zh.md (hunk 2 above)
Comment (derrickf87) [reply projects/cycan/README_zh.md:14]: Added WASM related content.

**Review by santryhuang — APPROVED** [review]

**Pull request merged**